### DISCUSSION TITLE
Post Preservation on old browser issues - change hardcoded js strategy

### DISCUSSION THREAD

**Discussion by spiazzi** [OpenIDC/mod_auth_openidc#1106] [Q&A]
Hi all,

During post preservation there is an issue with IE version < 8 (unfortunately till present for some users and use cases),.as these browser do not support JSON.stringify natively,and a polyfill JS like "json2.js"  should be included - JSON-js/json2.js at master · douglascrockford/JSON-js · GitHub.

In particular in this part of the code:
https://github.com/OpenIDC/mod_auth_openidc/blob/34fa5d79bd713f52b103f39907b5a81ac453a626/src/mod_auth_openidc.c#L461

There is a static JavaScript in the code hardcoded.
One option could be to change this code to include a JavaScript code taken from a configuration directive, that could point to a default js file present on Apache folder.

Another option could be to return a template html and this is either configurable from user with a default page in Apache folders or configuration file.

In general this hardcoded JavaScript seems not flexible and an option to overwrite I think meet to be found.

**Comment by zandbelt** [comment]
yes, there's #195 to track that
